pkgs/by-name/mi/mingtest/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  stdenv,
+  cmake,
+  texliveSmall,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  name = "mingtest";
+  version = "0.1.9-4";
```

Comment:
What does this `-4` come from?

Reply:
Maybe the 4 commit after the release ?

Reply:
Yes, 0.1.9 is the same version as on the git repository and the "-<number>" are regarding to changes in packages.nix.

Reply:
You also from Arch Linux? We don't add that here, just upstream version

Reply:
Please just remove it, it's unnecessary to do this.